python/lsst/analysis/tools/actions/plot/focalPlanePlot.py

```diff
@@ -487,8 +503,23 @@ def makePlot(
         bbox = dict(facecolor="paleturquoise", alpha=0.5, edgecolor="none")
         ax.text(0.8, 0.91, statsText, transform=fig.transFigure, fontsize=8, bbox=bbox)
 
-        patchCollection = PatchCollection(patches, alpha=0.4, edgecolor="black")
-        patchCollection.set_array(values)
+        # Defaults to med + 4 sigma Mad to match
+        # the camera team plots
+        if self.vmin:
+            vmin = self.vmin
+        else:
+            vmin = statMed - 4.0 * statMad
+        if self.vmax:
+            vmax = self.vmax
+        else:
+            vmax = statMed + 4.0 * statMad
+
+        valuesPlot = np.clip(values, vmin, vmax)
+
+        patchCollection = PatchCollection(
+            patches, edgecolor="white", cmap=cmap, linewidth=0.5, linestyle=(0, (0.5, 3))
```

Comment:
would it be too much to make these options (e.g., edgecolor, linewidth...) configurable too, or is it not worth it?

Reply:
I would prefer not to make all of these things configurable. I would like the plots produced to be uniform and not clutter up the code with too many config options.